Assets/Scripts/CreatureController.cs

```diff
@@ -19,9 +23,28 @@ public class CharacterController : MonoBehaviour {
         }
     }
 
-    // Purely for testing purposes
     void OnMouseUp () {
-        TakeDamage (20);
+        isSelected = !isSelected;
+
+        if (isSelected) {
+            // Set this creature as the target
+            combatManager.SetTarget(this);
+
+            ShowTargetingArrow();
+        } else {
+            // Reset the target if clicked again
+            combatManager.SetTarget(null);
+
+            HideTargetingArrow();
+        }
+    }
+
+    void ShowTargetingArrow() {
+        targetingArrow.SetActive(true);
+    }
+
+    void HideTargetingArrow() {
+        targetingArrow.SetActive(false);
     }
```

Comment:
If this is the same call except with a different const bool value would it be better to do something like this instead:
```suggestion
    void setTargetingArrow(bool isSelected){
                targetingArrow.SetActive(isSelected);
    }
```

Reply:
we may not even need a wrapper method for this because it's literally just a GameObject.call() but thats up2u

Reply:
Refactored to one method and moved the calls up to Combat Manager.